File: baekjoon/bronze/level2/test_1919/java/Main.java
package baekjoon.bronze.level2.test_1919.java;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        char[] a = sc.nextLine().toCharArray();
        char[] b = sc.nextLine().toCharArray();

        int[] aAlphabet = new int[26];
        int[] bAlphabet = new int[26];

        for (char c : a) {
            aAlphabet[c - 'a']++;
        }

        for (char c : b) {
            bAlphabet[c - 'a']++;
        }

        int sum = 0;
        for (int i = 0; i < 26; i++) {
            if (aAlphabet[i] != bAlphabet[i]) {
                sum += Math.abs(aAlphabet[i] - bAlphabet[i]);
            }
        }

        System.out.print(sum);
    }
}

File: baekjoon/bronze/level2/test_1919/kotlin/Main.kt
package baekjoon.bronze.level2.test_1919.kotlin

import kotlin.math.abs

fun main() {
    val a = readln().toCharArray()
    val b = readln().toCharArray()

    val alphabetA = Array(26) { 0 }
    val alphabetB = Array(26) { 0 }
    for (i in a) {
        alphabetA[i - 'a'] += 1
    }

    for (i in b) {
        alphabetB[i - 'a'] += 1
    }

    var count = 0
    for (i in 0..25) {
        count += abs(alphabetA[i] - alphabetB[i])
    }

    print(count)
}
